v0/destinations/am/transform.js

```diff
@@ -161,21 +164,43 @@ function updateConfigProperty(
             }
           }
         } else {
-          // get the destKey/outKey value from calling the util function
-          set(payload, outKey, AMUtils[funcName](message, sourceKey, Config));
+          const data = get(message.traits, outKey); // when in identify(or any other call) it checks whether outKey is present in traits
+          // then that value is assigned else function is applied.
+          // that key (outKey) will be a default key for wh actions and thus removed from the payload.
+          if (isDefinedAndNotNull(data)) {
```

Comment:
Do you think it makes sense to add the `mappedToDestination` check here because this change is specifically for VDM destinations?

Reply:
it may go to the other if else statements too